ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2454,7 +2454,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     }
                     break;
                 case StrategyTradeWorkFlowState.GoLongPlaceStopsPending:
-                    if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)
+                    if (IsStrategyUnSafeMode)

```

Comment:
This is required -  IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode  use the same paths a lot of the time-